app/helpers/adhoq/application_helper.rb

```diff
@@ -17,7 +17,7 @@ def schema_version
         ActiveRecord::SchemaMigration.maximum(:version)
       else
```

Comment:
> If the adhoq need to support Rails 3.2, it should be reverted the commit.

Thank you for your check. Adhoq droped support Rails3.2 last year. https://github.com/esminc/adhoq/pull/119